Source/JavaScriptCore/API/JSValueRef.h

```diff
@@ -88,6 +92,21 @@ typedef enum {
     kJSTypedArrayTypeBigUint64Array,
 } JSTypedArrayType JSC_API_AVAILABLE(macos(10.12), ios(10.0));
 
+/*!
+@enum JSRelationCondition
+@abstract     A constant identifying the type of JavaScript relation condition.
+@constant     kJSRelationConditionUndefined    Fail to compare two operands.
+@constant     kJSRelationConditionEqual        Two operands have equivalent values.
+@constant     kJSRelationConditionGreaterThan  The left operand is greater than the right operand.
+@constant     kJSRelationConditionLessThan     The left operand is less than the right operand.
+*/
+typedef enum {
+    kJSRelationConditionUndefined JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA)),
+    kJSRelationConditionEqual JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA)),
+    kJSRelationConditionGreaterThan JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA)),
+    kJSRelationConditionLessThan JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA))
```

Comment:
I don't think you need the `JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA)),` since it's on the enum as a whole. You only need it if you added new cases after the initial declaration.